app-containers/kubectl/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME="kubectl"
+PKGDES="kubernetes cli"
```

Comment:
You can make it better. Try harder.

Reply:
A command-line tool to manage Kubernetes instances and resources